collectors/lib/java.py

```diff
@@ -30,20 +30,19 @@
 
 PERIOD = 60
 
-IGNORED_METRICS = set(["revision", "hdfsUser", "hdfsDate", "hdfsUrl", "date",
-                       "hdfsRevision", "user", "hdfsVersion", "url", "version",
-                       "NamenodeAddress", "Version", "RpcPort", "HttpPort",
-                       # These are useless as-is because they represent the
-                       # thread that's dedicated to serving JMX RPCs.
-                       "CurrentThreadCpuTime", "CurrentThreadUserTime",
-                       # List of directories used by the DataNode.
-                       "StorageInfo",
-                       "VolumeInfo",
-                       "ServerName",
-                       "ZookeeperQuorum",
-                       "Verbose",
-                       "Type"
-                      ])
+IGNORED_METRICS = {"revision", "hdfsUser", "hdfsDate", "hdfsUrl", "date",
```

Comment:
I was seeing this as a set of commonly ignored metrics -- you're right though, there's some hbase/hdfs specific metrics in there that shouldn't. Let me look to see where those are being pulled in. If they're only used by the flume and kafka collectors, I'll move them to JmxMonitor